client/src/Views/Landing/gmailAuth.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Contains functions involved in checking and getting gmail authorisation for user
+ */
+
+const axios = require("axios");
+
+// TO DO: Get from .env
+const SERVER_ADDRESS = "http://localhost:3000";
+
+/*
+ * Check if current user has given permission for Mail Sender to access their
+ * gmail account
+ */
+exports.checkForGmailToken = async (userId) => {
```

Comment:
These exports won't work because they're using the CommonJS syntax from Node.js. Although we are using CommonJS on the back-end, on the front-end we should use the ES6 `import/export` syntax (see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)

Reply:
Thanks for the heads up! Begs the question why they are working though? Are they being transpiled or are they working ineffectively?